victorops-twilio.js

```diff
@@ -544,7 +545,7 @@ function buildOnCallList (twiml, context, payload) {
     // Creates a list of phone numbers based on the first 3 escalation policies
     const escPolicyUrlArray = createEscPolicyUrls(context, teamsArray[0].slug);
     const phoneNumberArray = escPolicyUrlArray.map(url => getPhoneNumbers(context, url, teamsArray[0].name, teamsArray[0].escPolicyName));
-
+    phoneNumberArray.length = 1;
```

Comment:
delete this, it was added so we wouldn't have to wait through multiple calls for testing

Reply:
updated